packages/wallet-ui/src/components/ui/molecule/TransactionsList/TransactionListItem/types.ts

```diff
@@ -17,10 +24,30 @@ export const getIcon = (transactionName: string): IconProp => {
   }
 };
 
+export const getTranslationNameForTxnType = (
+  txnType: TxnType,
+  translate: (key: string) => string,
+): string => {
+  switch (txnType) {
+    case TxnType.Send:
+      return translate('send');
+    case TxnType.UpgradeAccount:
+      return translate('upgradeAccount');
+    case TxnType.ContractInteraction:
+      return translate('contractInteraction');
+    case TxnType.Deploy:
+      return translate('deploy');
+    case TxnType.DeployAccount:
+      return translate('deployAccount');
+    default:
+      return translate('unknown');
+  }
+};
+
 export const getTxnName = (
```

Comment:
we are return the type enum
```suggestion
export const getTxnType = (
```